components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java

```diff
@@ -752,7 +752,9 @@ public APIDefinitionValidationResponse validateAPIDefinition(String apiDefinitio
             }
             OASParserUtil.updateValidationResponseAsSuccess(
                     validationResponse, apiDefinition, openAPI.getOpenapi(),
-                    info.getTitle(), info.getVersion(), info.getTitle().replaceAll("\\s","").toLowerCase(),
+                    info.getTitle(), info.getVersion(),
+                    (info.getTitle() == null || info.getTitle().isEmpty() ? null :
```

Comment:
We can use StringUtil.IsBlank.. this will check null, empty, blank spaces as well

Reply:
Fixed with ec443fd3bf008bac84e91c3966e6f80bdc8413cf